packages/flutter_markdown/lib/src/syntax_highlighter.dart

```diff
@@ -0,0 +1,339 @@
+part of flutter_markdown;
+
+class SyntaxHighlighterStyle {
+  SyntaxHighlighterStyle({
+    this.baseStyle,
+    this.numberStyle,
+    this.commentStyle,
+    this.keywordStyle,
+    this.stringStyle,
+    this.punctuationStyle,
+    this.classStyle,
+    this.constantStyle
+  });
+
+  static SyntaxHighlighterStyle defaultStyle() {
+    return new SyntaxHighlighterStyle(
+      baseStyle: new TextStyle(color: Colors.black),
+      numberStyle: new TextStyle(color: Colors.blue[800]),
+      commentStyle: new TextStyle(color: Colors.grey[500]),
+      keywordStyle: new TextStyle(color: Colors.purple[500]),
+      stringStyle: new TextStyle(color: Colors.green[600]),
+      punctuationStyle: new TextStyle(color: Colors.black),
+      classStyle: new TextStyle(color: Colors.deepPurple[700]),
+      constantStyle: new TextStyle(color: Colors.brown[500])
```

Comment:
These shouldn't depend on material.dart.  You should just define your own set of color constants.